pkg/kn/commands/source/apiserver/describe.go

```diff
@@ -18,23 +18,34 @@ import (
 	"errors"
 	"fmt"
 	"sort"
+	"strings"
 
 	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
 	v1alpha2 "knative.dev/eventing/pkg/apis/sources/v1alpha2"
 
 	"knative.dev/client/lib/printing"
 	"knative.dev/client/pkg/kn/commands"
 	"knative.dev/client/pkg/printers"
 )
 
+var describeExample = `
+  # Describe an ApiServer source with name 'k8sevents'
+  kn source apiserver describe k8sevents
+
+  # Describe an ApiServer source with name 'k8sevents' in YAML format
+  kn source apiserver describe k8sevents -o yaml`
```

Comment:
```suggestion
  # Describe an api-server source with name 'k8sevents'
  kn source apiserver describe k8sevents

  # Describe an api-server source with name 'k8sevents' in YAML format
  kn source apiserver describe k8sevents -o yaml`
```

Reply:
(make sure to run `./hack/build.sh` after these changes so same could be reflected in .md docs)

Reply:
Done